### PR TITLE
Fix the multi-class support in a cleaner way.

### PR DESCRIPTION
**Pull request by SuX97**
Move multi-class and stuff to **kwargs, do not need to add it once a new head appears.

### CODE REVIEW

File: mmaction/models/heads/slowfast_head.py [hunk 1]
```diff
@@ -26,9 +28,10 @@ def __init__(self,
                  loss_cls=dict(type='CrossEntropyLoss'),
                  spatial_type='avg',
                  dropout_ratio=0.8,
-                 init_std=0.01):
+                 init_std=0.01,
+                 **kwargs):
 
-        super().__init__(num_classes, in_channels, loss_cls)
+        super().__init__(num_classes, in_channels, loss_cls, kwargs)
```
Comment (innerlee) [inline mmaction/models/heads/slowfast_head.py:34]: **

**Comment by innerlee** [comment]
has the multi-class option been tested?

**Comment by SuX97** [comment]
> has the multi-class option been tested?

[Here](https://github.com/open-mmlab/mmaction2/blob/master/tests/test_models/test_head.py#L119). Only tested on TSNHead, but I think it will work as long as all heads are written in the same way.

**Comment by innerlee** [comment]
fair

**Review by innerlee — APPROVED** [review]

**Pull request merged**